website/client/js/controllers/inventoryCtrl.js

```diff
@@ -54,11 +54,16 @@ habitrpg.controller("InventoryCtrl",
 
     $scope.$watch('user.items.gear', function(gear){
       $scope.gear = {};
+      $scope.gearByType = {};
       _.each(gear.owned, function(v,key){
         if (v === false) return;
         var item = Content.gear.flat[key];
-        if (!$scope.gear[item.klass]) $scope.gear[item.klass] = [];
+        if (!$scope.gear[item.klass])
```

Comment:
I simply moved it to a new line for clarity but left out the brackets simply in favor of leaving it alone because I don't know what the convention is for this repository. I'm fine either way.